source/change-streams/tests/unified/change-streams-disambiguatedPaths.yml

```diff
@@ -1,5 +1,5 @@
 description: "disambiguatedPaths"
-schemaVersion: "1.3"
+schemaVersion: "1.4"
```

Comment:
`serverless` is not a supported `runOnRequirement` field for schemes with version < 1.4 .